ChangeLog.md

```diff
@@ -1,3 +1,7 @@
+# v1.1.2.0
```

Comment:
Please don't specify the version number until we're making the release PR; t creates more work if we decide to make a breaking change or rename the version for PR purposes 🧐